UPGRADE-2.0.md

```diff
@@ -35,3 +35,7 @@
     
         * `sylius.mongodb_odm.repository.class`
         * `sylius.phpcr_odm.repository.class`
+
+    * The follwing configuration options were removed:
```

Comment:
* The **following** configuration options were removed:

Reply:
Thanks for catching